spot_wrapper/calibration/calibration_util.py

```diff
@@ -176,6 +177,10 @@ def get_multiple_perspective_camera_calibration_dataset(
             origin_t_planning_frame=primed_pose,
             duration_sec=0.1,
         )
+        # base_to_hand1 = initial_pose
```

Comment:
Do you mind deleting these ocmments if they're not required for something important?